source/image-handler/index.js

```diff
@@ -22,9 +22,14 @@ exports.handler = async (event) => {
         const request = await imageRequest.setup(event);
         console.log(request);
         const processedRequest = await imageHandler.process(request);
+        const headers = getResponseHeaders();
+        headers["Content-Type"] = request.ContentType;
+        headers["Expires"] = request.Expires;
+        headers["Last-Modified"] = request.LastModified;
+        headers["Cache-Control"] = request.CacheControl;
```

Comment:
Would it be possible to supply overridden values? In the base64 encoded string
{
"header": [{'Expires' : x }]
}

Reply:
Good catch, @bretto36. I'm not using either of those features (changing formats or base64), but I'll take a look when I get a chance.